core/drivers/crypto/crypto_api/include/drvcrypt_acipher.h

```diff
@@ -133,6 +133,17 @@ struct drvcrypt_secret_data {
 	struct drvcrypt_buf secret; /* Shared secret */
 };
 
+/*
+ * Cipher data
+ */
+struct drvcrypt_cipher_data {
+	uint32_t algo;                  /* Operation algorithm */
+	void *key;                      /* Public or Private Key */
+	size_t size_sec;                /* Security size in bytes */
```

Comment:
prefer `size_t key_size; /* Private key size in bytes */`

Reply:
In drvcrypt_sign_data,drvcrypt_secret_data and other structs, size_sec is used.I wonder if it would be better to keep correspondence with them.

Reply:
Ok.